lua/obsidian/command.lua

```diff
@@ -1,5 +1,3 @@
-local Path = require "plenary.path"
-local Job = require "plenary.job"
```

Comment:
I'm guessing you didn't mean to remove these?

Reply:
Strange I don't really know what happened